x-pack/legacy/plugins/transform/index.ts

```diff
@@ -4,19 +4,9 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { resolve } from 'path';
-
-import { PLUGIN } from './common/constants';
-
 export function transform(kibana: any) {
```

Comment:
@joshdover  is this `index.ts` file still needed after cutover to `x-pack/plugins`?

Reply:
Maybe @sebelga can also chime in on this one. I based that off the structure of the `index_management` plugin which also has this one file left in `legacy`.

Reply:
I had to leave that file in the legacy folder so dependant plugins still in the legacy world that read the configuration would not complain.

If there aren't any plugin the the "legacy" folder that read the transform plugin config, this file should not be needed.